src/models/stores/ui.ts

```diff
@@ -47,78 +38,27 @@ export const UIDialogModel = types
 type UIDialogModelSnapshot = SnapshotIn<typeof UIDialogModel>;
 type UIDialogModelSnapshotWithoutType = Omit<UIDialogModelSnapshot, "type">;
 
-// This generic model should work for both the problem tab, and the MyWork/ClassWork tabs
-// The StudentWorkspaces tab might work this way too. It has an open group which could be
-// stored in the openSubTab. And then it might have an open'd student document of the group
-// this would be the openDocument.
-export const UITabModel = types
-  .model("UITab", {
-    id: types.identifier,
-    openSubTab: types.maybe(types.string),
-    // The key of this map is the sub tab label
-    openDocuments: types.map(types.string),
-    openSecondaryDocuments: types.map(types.string)
-  });
-
 export const UIModel = types
   .model("UI", {
     annotationMode: types.maybe(types.string),
-    dividerPosition: kDividerHalf,
     error: types.maybeNull(types.string),
-    activeNavTab: ENavTab.kProblems,
     selectedTileIds: types.array(types.string),
     selectedCommentId: types.maybe(types.string),
     scrollTo: types.maybe(ScrollToModel),
-    showAnnotations: true,
     showDemo: false,
     showDemoCreator: false,
-    showTeacherContent: true,
-    showChatPanel: false,
     dialog: types.maybe(UIDialogModel),
-    tabs: types.map(UITabModel),
-    problemWorkspace: WorkspaceModel,
     learningLogWorkspace: WorkspaceModel,
-    teacherPanelKey: types.maybe(types.string),
     dragId: types.maybe(types.string) // The id of the object being dragged. Used with dnd-kit dragging.
   })
   .volatile(self => ({
     defaultLeftNavExpanded: false,
-    problemPath: ""
+    problemPath: "" //TODO: figure out if we need problem path
```

Comment:
This doesn't seem to be referenced anymore outside of this file. 
And setProblemPath is also not referenced.
So it seems safe to remove both.